centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -18093,6 +18090,33 @@ msgstr "Valeur définie par la métrique"
 msgid "The {{module}} license will expire in {{expirationDate}} days"
 msgstr "La license {{module}} expirera dans {{expirationDate}} jours"
 
+msgid "Last hour"
+msgstr "La dernière heure"
+
+msgid "Last 24 hours"
+msgstr "Les dernières 24 heures"
+
+msgid "Last 7 days"
+msgstr "Les 7 derniers jours"
+
+msgid "Last 30 days"
+msgstr "Les 30 derniers jours"
+
+msgid "Last 3 months"
+msgstr "Les 3 derniers mois"
+
+msgid "Last 12 months"
+msgstr "Les 12 derniers mois"
+
+msgid "Start"
+msgstr "Début"
+
+msgid "End"
+msgstr "Fin"
+
+msgid "Value customized"
```

Comment:
```suggestion
msgid "Custom value"
```

Reply:
This is displayed when a value is customized like so "Critical threshold: 60%. Value customized"